check.py

```diff
@@ -195,21 +195,14 @@ def fail_if_not_contained(actual, expected):
           raise Exception('wasm interpreter error: ' + err) # failed to pretty-print
         raise Exception('wasm interpreter error')
 
-print '\n[ checking binaryen-shell... ]\n'
```

Comment:
But this tested the contents of what the user can read when `--help` is requested. We don't check that anywhere else, do we?